app/code/Magento/PageBuilder/Model/Stage/HtmlFilter.php

```diff
@@ -98,7 +98,7 @@ public function filterHtml(string $content): string
      */
     private function getChildrenInnerHtml(\DOMElement $element): string
     {
-        $innerHTML = '';
+        $innerText = '';
```

Comment:
Please elaborate on the purpose of this change. 

Also as I can see here you are just changing the variable name. If that is so then please update the same in line number 115 and 118:

https://github.com/magento/magento2-page-builder/blob/1a52ef4c5fe7dc2daf8504f6e701ad64011c6457/app/code/Magento/PageBuilder/Model/Stage/HtmlFilter.php#L115

https://github.com/magento/magento2-page-builder/blob/1a52ef4c5fe7dc2daf8504f6e701ad64011c6457/app/code/Magento/PageBuilder/Model/Stage/HtmlFilter.php#L118

Reply:
Hii @engcom-Hotel  Thanks For Reviewing Yes done this changes

Reply:
Also please help us with the purpose of this PR. In the actual codebase we are not using the `innerHTML` property, here `$innerHTML` is used as a variable. So this change doesn’t make any sense to me.

Please correct me if I am wrong.